### DISCUSSION TITLE
Datenbedeutung

### DISCUSSION THREAD

**Discussion by Ingo-Vetter** [nathanmarlor/foxess_modbus#616] [General]
Hallo zusammen, 

ich habe ein Verständnisproblem bei den Daten

sensor.wr_master_total_yield_today
und 
sensor.wr_master_total_yield_total

Was sind das für Daten, bzw. wie kann ich diese Daten verstehen?

Viele Grüße
Ingo

**Comment by FozzieUK** [comment]
The total yield values are the least useful of all the statistics the inverter produces - the total yield is the output from the inverter - in effect all of the power that has flowed through the inverter using either battery or solar.
